data.sql

```diff
@@ -2,3 +2,13 @@ INSERT INTO animals(pet_name, data_of_birth, escape_attempts, neutered, weight_k
 INSERT INTO animals(pet_name, data_of_birth, escape_attempts, neutered, weight_kg) VALUES('Gabumon', TO_DATE('11/15/2018', 'MM/DD/YYYY'), 2, TRUE, 8.0);
 INSERT INTO animals(pet_name, data_of_birth, escape_attempts, neutered, weight_kg) VALUES('Pikachu', TO_DATE('01/07/2021', 'MM/DD/YYYY'), 1, FALSE, 15.04);
 INSERT INTO animals(pet_name, data_of_birth, escape_attempts, neutered, weight_kg) VALUES('Devimon', TO_DATE('05/12/2017', 'MM/DD/YYYY'), 5, TRUE, 11.0);
+
+
```

Comment:
![Capture2](https://user-images.githubusercontent.com/93039359/206019305-ee3d7bf1-bce2-47dc-856e-0a0a7ca4593d.PNG)

- [ ] You didn't add this query in your schema.sql file. Kindly add it 😊

Reply:
Thank you @HAMMAS-SALEEM I have completed this step ✔️